### PR TITLE
startTime, endTime and orderId extra notes for allOrders endpoint

### PR DESCRIPTION
**Pull request by mpetrinidev**
I've added some extra notes for the `allOrders` endpoint.

I received the following error when I sent the same value for startTime and endTime parameters.

> Code: 1102  
> Message: Mandatory parameter 'endTime' was not sent, was empty/null, or malformed.

endTime has to be greater than startTime (which makes sense, but the error reporting is not very clear)

--- 

And the second note is related to when you used either orderId or (startTime and endTime) optional parameters.

These notes can be helpful to avoid extra calls to Binance API.

### CODE REVIEW

**Review by alisscco — APPROVED** [review]

File: rest-api.md [hunk 1]
```diff
@@ -1310,6 +1310,8 @@ timestamp | LONG | YES |
 **Notes:**
 * If `orderId` is set, it will get orders >= that `orderId`. Otherwise most recent orders are returned.
 * For some historical orders `cummulativeQuoteQty` will be < 0, meaning the data is not available at this time.
+* If `startTime` and `endTime` are set, `endTime` has to be greater than `startTime`. 
+* Either `orderId` or (`startTime` and `endTime`) must be sent when used these optional parameters.
```
Comment (2pd) [inline rest-api.md:1314]: This is not correct as `orderId` or (`startTime` and `endTime`) can be omitted.

**Review by 2pd — CHANGES_REQUESTED** [review]

File: rest-api.md (hunk 1 above)
Comment (adderall-prozac) [reply rest-api.md:1314]: We'll add a clarification on this though for future reference though. Thank you.

**Comment by alisscco** [comment]
Noted ...
Please if I may ask you to specify which nodes?
For which pool or all ?
Or just binance ?
Sorry for the ignorance in this. ,
Regards,

File: rest-api.md (hunk 1 above)
Comment (mpetrinidev) [reply rest-api.md:1314]: I've meant if you only sent either `orderId` or (`startTime` and `endTime`). You cannot use all parameters together.

**Comment by alisscco** [comment]
On it ..

**Comment by 2pd** [comment]
We have made the change at #39 , thanks for feedback.

File: rest-api.md (hunk 1 above)
Comment (alisscco) [reply rest-api.md:1314]: Trying to figure it out .. I didn't do it on purpose ..

Comment (alisscco) [reply rest-api.md:1314]: Trying to figure it out .. I didn't do it on purpose ..

**Pull request closed without merging**